src/TagDependencyTrait.php

```diff
@@ -42,7 +43,21 @@ public static function commonTag()
     public function objectTag()
     {
         /** @var \yii\db\ActiveRecord $this */
-        return NamingHelper::getObjectTag($this->className(), $this->getPrimaryKey());
+        $primary_key;
+        if (count($this->primaryKey()) == 1)
+        {
+            $key = $this->primaryKey()[0];
+            $primary_key = isset($oldfields[$key]) ? $oldfields[$key] : $this->$key;
```

Comment:
This is my bad, should be an optional arg to objectTag, I made the commit using the github website, so just an oversight.